IndentationError in model.py

Python 3 treats IndentationErrors as syntax errors

flake8 testing of https://github.com/wannabeOG/Mask-RCNN on Python 3.6.3

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./model.py:1255:4: E999 IndentationError: unindent does not match any outer indentation level
   .
   ^
1     E999 IndentationError: unindent does not match any outer indentation level
1
```

--- a/model.py
+++ b/model.py
@@ -1252,7 +1252,6 @@ def build_rpn_targets(image_shape, anchors, gt_class_ids, gt_boxes, config):
     # Compute overlaps [num_anchors, num_gt_boxes]
     overlaps = utils.compute_overlaps(anchors, gt_boxes)
 
-   .
     anchor_iou_argmax = np.argmax(overlaps, axis=1)
     anchor_iou_max = overlaps[np.arange(overlaps.shape[0]), anchor_iou_argmax]
     rpn_match[(anchor_iou_max < 0.3) & (no_crowd_bool)] = -1

